steps/lifx-notifier/1.2.1/step.yml

```diff
@@ -0,0 +1,105 @@
+title: LIFX Notifier
+summary: Sends signals to your LIFX bulb during builds.
+description: |-
+  Add this step to your workflow when color change is required.
+
+  This step can be used at the start (notify build start) and at the end (notify success/failure) of your workflow.
+  If there's no need to notify about workflow start than adding step to the end of the workflow will be enough.
+website: https://github.com/ShotSkydiver/steps-lifx-notifier
+source_code_url: https://github.com/ShotSkydiver/steps-lifx-notifier
+support_url: https://github.com/ShotSkydiver/steps-lifx-notifier/issues
+published_at: 2021-05-21T16:03:58.500425-06:00
+source:
+  git: https://github.com/ShotSkydiver/steps-lifx-notifier.git
+  commit: 472646e717257221e3eba2bad03f7f6d5af0a00f
+host_os_tags:
+- osx-10.8
+- osx-10.10
+- ubuntu
+type_tags:
+- utility
+- notification
+is_requires_admin_user: false
+is_always_run: true
+is_skippable: true
+run_if: ""
+inputs:
+- auth_token: null
+  opts:
+    description: |
+      This step uses official LIFX API,
+      which requires OAuth2 Access token (https://api.developer.lifx.com/docs/authentication).
+    is_required: true
```

Comment:
This input should be marked as sensitive

Reply:
+1 on this, it is a requirement for new steps. I strongly urge you to add this change to this PR (but otherwise since this particular code is not changed, I've approved the PR itself)